components/HomePage/UserHomePage.tsx

```diff
@@ -1,12 +1,59 @@
-import React from "react";
+import React, { useState, useContext } from "react";
 import { UserCalendar } from "../Calendar/UserCalendar";
+import { AvailabilityModal } from "../ManageAvailabilityWizard/AvailabilityModal";
+import styles from "./AdminHomePage.module.css"; //TODO change to user specific
+import useSWR from "swr";
+import { Loader } from "../util/Loader";
+import { Error } from "../util/Error";
+import { APIContext } from "../../context/APIContext";
 
 type UserHomeProp = {
   userId: string;
 };
 
 const UserHomePage: React.FC<UserHomeProp> = ({ userId }) => {
-  return <UserCalendar userId={userId} />;
+  const [showManageAvailability, setShowManageAvailability] = useState(false);
+  const client = useContext(APIContext);
+  const { data, mutate, error } = useSWR(`api/availability/${userId}`, () => client.getAvailabilities(userId));
+  if (error) return <Error />;
+  if (!data) return <Loader />;
+
+  const handleClose = (): void => {
+    setShowManageAvailability(false);
+  };
+
+  const availability = {
+    "Mon": (data.mon == null) ? [] : data.mon,
+    "Tue": (data.tue == null) ? [] : data.tue,
+    "Wed": (data.wed == null) ? [] : data.wed,
+    "Thu": (data.thu == null) ? [] : data.thu,
+    "Fri": (data.fri == null) ? [] : data.fri,
+    "Sat": (data.sat == null) ? [] : data.sat,
+  }
+  const timeZone = data.timeZone
```

Comment:
we need to render the availibilitiyes by timzeone. Where this gets interesting is when things cross the date boundary.  We can use luxon to do it though. I would take all the times and give them each a dummy date. Then convert the timezone to the correct one (you dont need to do any of this if the timezone specified is the same as the current timezone). Then, see if the date got shifted (and if so, but how much. It should be at most +-1 day). If it was shifted, move the time to the correct day